ingest-app/docs/api.md

```diff
@@ -698,18 +702,59 @@ Example output:
 
 ```
 {
```

Comment:
I don't feel strong about this. Just my 2 cents: It is probably not necessary to provide the exact input and output here. It is too excessive and overwhelms the real useful info in the API doc. We may want to just drop the full input and output from the API doc.

Reply:
for consistency I think it is needed, however the ISO example is a bit long. What if we did another format that is not as verbose? The other idea I had was to add vertical scroll bars to the rendering CSS